Makefile

```diff
@@ -128,12 +128,13 @@ ifneq ($(SKIP_TESTS), true)
 endif
 
 .PHONY: test-scorecard
-test-scorecard: check_cert_manager
+test-scorecard: check_cert_manager kustomize
```

Comment:
I am guessing downstream build will remove `check_cert_manager` since it would try to install within tests?

Reply:
It doesn't use this target at all.